src/utilities/inputProcessors.ts

```diff
@@ -149,6 +150,20 @@ export const processInputReleaseBranches = (): BranchSpec[] | undefined => {
   return validateInputReleaseBranches(parsedInput);
 };
 
+export const processInputConfigFile = (): string | undefined => {
```

Comment:
```suggestion
export const processInputConfigFile = (): string | void => {
```

Reply:
This would yield:
`void is only valid as a return type or generic type variable(@typescript-eslint/no-invalid-void-type)`